pymodbus/datastore/store.py

```diff
@@ -72,16 +72,12 @@ def default(self, count, value=False):
         :param value: The default value to set to the fields
         """
         self.default_value = value  # pylint: disable=attribute-defined-outside-init
-        self.values = [  # pylint: disable=attribute-defined-outside-init
-            self.default_value
-        ] * count
+        self.values = [self.default_value] * count
```

Comment:
This does not seem right, and you missed the previous disable !

The problem is that the base class misses an __init__ where the variables should be defined.

Reply:
In this case I followed `pylint` blindly.  It warned me the disable was unnecessary.  I will revisit after rethinking `BaseModbusDataBlock(dict)`